src/main/java/gregtech/common/command/benchmark/CommandBenchmark.java

```diff
@@ -0,0 +1,23 @@
+package gregtech.common.command.benchmark;
+
+import net.minecraft.command.ICommandSender;
+import net.minecraftforge.server.command.CommandTreeBase;
+
+import org.jetbrains.annotations.NotNull;
+
+public class CommandBenchmark extends CommandTreeBase {
+
+    public CommandBenchmark() {
```

Comment:
Do we want to restrict permission level? Not sure if we want just anyone to be able to run this.